src/common/Timer.h

```diff
@@ -114,14 +114,12 @@ class StopWatch {
         if (getrusage(RUSAGE_SELF, &tmp_rusage) == 0) {
             time_start = OSMTTimeVal(tmp_rusage);
         }
-        else assert(false);
     }
     ~StopWatch()
     {
         if (getrusage(RUSAGE_SELF, &tmp_rusage) == 0) {
             timer += OSMTTimeVal(tmp_rusage) - time_start;
         }
-        else assert(false);
```

Comment:
Why remove these `asserts`? If you are removing them you do not need to include the header, no?

Reply:
It causes a warning to the destructor to have a implicit non-throwing exception specification